charts/infra-apps/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: infra-apps
 description: Argo CD app-of-apps config for infrastructure components
 type: application
-version: 0.164.0
+version: 0.165.0
```

Comment:
shouldn't we do patchfix?

Reply:
I decided against it, as a new minor release better communicates that it's a new version.